### PR TITLE
New version: SMPlayer.SMPlayer version 23.12.0

### LINKED ISSUE #133923: [Update Request]: SMPlayer.SMPlayer 23.12.0
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

SMPlayer.SMPlayer

### Package Version

23.12.0

### Please describe the changes you would like to see

https://github.com/smplayer-dev/smplayer/releases/tag/v23.12.0

Changelog:  

    Implemented support for the new Opensubtitles API.
    Fix play/pause with mpv 0.37.
    Fix loading playlists with extension m3u8.
    Fix video jitter/shudder when clicking on timeline bar.
    Other small fixes.

### PR DESCRIPTION
**Pull request by Exorcism0666**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.0.0 :rocket:

- Resolve: #133923
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/133927)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by Exorcism0666** [comment]
@stephengillie help 😭

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-13T21:39:24.5084074Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[InstallerSwitches] Description: Value type not permitted by 'type' constraint. Error context: <root> Description: Failed to validate against schema associated with property name 'InstallerSwitches'. ] File: SMPlayer.SMPlayer.installer.yaml Manifest Error: Encountered unexpected root node. Manifest Warning: Silent and SilentWithProgress switches are not specified for InstallerType exe. Please make sure the installer can run unattended. Manifest Warning: Silent and SilentWithProgress switches are not specified for InstallerType exe. Please make sure the installer can run unattended.

(Automated response - build 567.)

File: manifests/s/SMPlayer/SMPlayer/23.12.0/SMPlayer.SMPlayer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Windows.Desktop
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+InstallerSwitches:
+- Silent: /S
+- SilentWithProgress: /S
```
Comment (stephengillie) [inline manifests/s/SMPlayer/SMPlayer/23.12.0/SMPlayer.SMPlayer.installer.yaml:16]:
```suggestion
InstallerSwitches:
  Silent: /S
  SilentWithProgress: /S
```
I'm not a YAML expert, but copying from my notes suggests that this shouldn't have the dashes ( - ).

**Review by stephengillie — APPROVED** [review]

**Pull request merged**